sopel/config/types.py

```diff
@@ -211,12 +211,45 @@ def configure(self, prompt, default, parent, section_name):
 class ListAttribute(BaseValidated):
     """A config attribute containing a list of string values.
 
-    Values are saved to the file as a comma-separated list. It does not
-    currently support commas within items in the list. By default, the spaces
-    before and after each item are stripped; you can override this by passing
-    ``strip=False``."""
+    From this :class:`StaticSection`::
 
-    ESCAPE_CHARACTER = '\\'
+        class SpamSection(StaticSection):
+            cheeses = ListAttribute('cheeses')
+
+    the option will be exposed as a Python :class:`list`::
+
+        >>> config.spam.cheeses
+        ['camembert', 'cheddar', 'reblochon']
+
+    which come from this configuration file::
+
+        [spam]
+        cheeses = camembert
```

Comment:
Given that the updated `configuration.rst` examples all place the first list value on the next line, shouldn't the examples in this file do the same?

Reply:
Hm. Probably, yes!